src/cinder/Surface.cpp

```diff
@@ -477,7 +477,7 @@ void SurfaceT<T>::copyRawRgbFullAlpha( const SurfaceT<T> &srcSurface, const Area
 	int32_t width = srcArea.getWidth();
 
 	for( int32_t y = 0; y < srcArea.getHeight(); ++y ) {
-		const T *src = reinterpret_cast<const T*>( reinterpret_cast<const uint8_t*>( srcSurface.getData() + srcArea.x1 * 4 ) + ( srcArea.y1 + y ) * srcRowBytes );
+		const T *src = reinterpret_cast<const T*>( reinterpret_cast<const uint8_t*>( srcSurface.getData() + srcArea.x1 * 3 ) + ( srcArea.y1 + y ) * srcRowBytes );
```

Comment:
Just wondering: to be consistent, should this be using `srcPixelInc` and `dstPixelInc` as some of the surrounding methods do? Or do those not give the right result here?